src/test/java/com/epam/tc/hw2/TestControlElementsOfElementsPage.java

```diff
@@ -0,0 +1,83 @@
+package com.epam.tc.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+
+public class TestControlElementsOfElementsPage extends AbstractTestPage {
+
+
+    private static final  String EXPECTED_PAGE_TITLE = "Home Page";
+    private static final String EXPECTED_USER_NAME = "ROMAN IOVLEV";
+    private static final Map<String, String> EXPECTED_LOG_NAME_STATUS = Map.of(
+        "Water", "true",
+        "Wind", "true",
+        "metal", "Selen",
+        "Colors", "Yellow");
+    private static final String EXPECTED_SELECT_COLOR = "Yellow";
+    private static final int ORDER_OF_NAME_WORD_IN_LOG = 1;
+    private static final int ORDER_OF_STATUS_WORD_IN_LOG = 5;
+
+
+    @BeforeClass
```

Comment:
You probably forgot to remove this block of code since you already have it in your base class

Reply:
Yes, my bad! Deleted.